gtfs/gtfs_utils/gtfs_utils/config.schema.json

```diff
@@ -0,0 +1,145 @@
+{
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://example.com/config.schema.json",
+    "title": "Main configuration object parameter",
+    "description": "The configuration for a gtfs_stats run",
+    "type": "object",
+    "properties": {
+      "files": {
+        "$ref": "#/definitions/files"
+      },
+      "s3": {
+        "$ref": "#/definitions/s3"
+      },
+      "date_range": {
+        "description": "First and last date of the gtfs files to be download from bucket. {Format: *YYYY-MM-DD*}",
```

Comment:
Add a note that this field is only relevant if `use_data_from_today` is `false`.